cilium/cmd/config.go

```diff
@@ -73,15 +75,42 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 			if err := command.PrintOutput(cfgStatus.Realized.Options); err != nil {
 				os.Exit(1)
 			}
+			if err := command.PrintOutput(cfgStatus.DaemonConfigurationMap); err != nil {
+				os.Exit(1)
```

Comment:
I realize that the existing code already did this, but we should really do better handling on these errors:

```suggestion
				Fatalf(err)
```